### PR TITLE
Add more information to the readme

### PR DESCRIPTION
**Pull request by jtraglia**
The README feels empty and I would like to add more information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,43 +29,56 @@ There are functions for KZG operations:
...
+### Why C?
 
-```
-cd src && make
-```
+The primary reason for choosing C is because
+[blst](https://github.com/supranational/blst), the BLS12-381 signature library
+we wanted to use, is mostly written in C. We were concerned that using blst with
+another language, like Rust, and then building bindings on top of it would
+introduce too much overhead.  Furthermore, the C toolchain is ubiquitous, and it
```
Comment (asn-d6) [inline README.md:82]: Given that the FFI overhead from rust is not really a concern anymore, perhaps you can remove the overhead sentence. Another valid positive of doing it in C is that it doesn't have the heavy Rust runtime (smaller memory/binary footprint).

**Comment by asn-d6** [comment]
This all looks great! Not merging yet since this is a draft.

File: README.md (hunk 1 above)
Comment (jtraglia) [reply README.md:82]: Will update this paragraph to reflect this 👍

**Comment by jtraglia** [comment]
@asn-d6 Alright, I think I'm happy with the current state of this. Ready for a final review.

Also, happy to add other remarks if you can think of any.

**Comment by ppopth** [comment]
LGTM!

**Pull request merged**